spring-cloud-aws-autoconfigure/src/test/java/io/awspring/cloud/autoconfigure/dynamodb/DynamoDbAutoConfigurationTest.java

```diff
@@ -61,34 +54,42 @@ void dynamoDBAutoConfigurationIsDisabled() {
 
 	@Test
 	void dynamoDBAutoConfigurationIsEnabled() {
-		this.contextRunner.withPropertyValues("spring.cloud.aws.dynamodb.enabled:true").run(context -> {
```

Comment:
Not necessarily related to these removed lines, but i am missing test where DAX client is not on the classpath. You can read about it here: https://spring.io/blog/2018/03/07/testing-auto-configurations-with-spring-boot-2-0 or search for other tests where we filter out classes.